File: Modul7/Soal3/PRAK603-2210817110008-Aditya Oktaviari.py
n1, n2= map(int, input().split())
angka1= [] 
angka2= []

if(n1 != n2):
    print("\nJumlah tidak sama")

else:
    nilai1= list(map(int, input().split()))
    nilai2= list(map(int, input().split()))
    print()
    for i in range(0, n1):
        angka1.append(nilai1[i])

    for i in range(0, n2):
        angka2.append(nilai2[i])
        
    for i in range(0, n1):
       print(angka1[i]*angka2[i], end=' ')
      

File: Modul7/Soal3/PRAK603-2210817110008-Aditya Oktaviari.c
#include <stdio.h>

int main()
{
    int n1, n2, isi, i;
    scanf("%d %d", &n1, &n2);
    int nilai1[n1], nilai2[n2];

    if(n1 != n2){
        printf("\nJumlah tidak sama");
    }

    else{
        for(i=0; i<n1; i++){
            scanf("%d", &isi);
            nilai1[i]=isi;
        }
        for(i=0; i<n2; i++){
            scanf("%d", &isi);
            nilai2[i]=isi;
        }
        printf("\n");
        for(i=0; i<n1; i++){
            printf("%d ", nilai1[i]*nilai2[i]);
        }
    }
    return 0;
}
